app/controllers/SubmitFormController.scala

```diff
@@ -136,12 +136,12 @@ class SubmitFormController @Inject()(mcc: MessagesControllerComponents,
                 )).addingToSession(SessionKeys.viewModel -> Json.toJson(viewModel).toString())
 
               } else {
-                logger.warn("[SubmitFormController][renderViewWithoutSessionData] " +
```

Comment:
same here

Reply:
this is not an application level error, its a journey error, so i think it can be changed to warnLog. i set it as errorLog because it terminates the journey and the user is sent to a bad request page